src/services/rpcService.ts

```diff
@@ -0,0 +1,134 @@
+import {
+    Connection,
+    PublicKey,
+  } from '@solana/web3.js';
+  import {
+    TOKEN_PROGRAM_ID,
+  } from '@solana/spl-token';
+  import { BN } from '@coral-xyz/anchor';
+  import { validatePublicKey } from '@/utils/publicKey';
+  
+  const DEVNET_RPC_ENDPOINT = 'https://api.devnet.solana.com';
+  const MAINNET_RPC_ENDPOINT = 'https://api.mainnet-beta.solana.com';
```

Comment:
These should be source from `.env`

Reply:
they're currently just the default. I'll update the name to make it clear